### PR TITLE
Dynamic Module + KVQueue

### PR DESCRIPTION
**Pull request by Sorikairox**

### CODE REVIEW

File: src/techniques/events.md [hunk 1]
```diff
@@ -37,7 +37,9 @@ To dispatch (i.e., fire) an event, first inject `EventEmitter` using standard co
 constructor(private eventEmitter: EventEmitter) {}
 ```
 
-> **Hint** 💡 Import the `EventEmitter` from the `danet/mod.ts` package.
+::: **Hint** 💡
```
Comment (marco-souza) [inline src/techniques/events.md:40]: Thanks for fixing it! 🙏🏼

File: src/techniques/kvQueue.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+---
+label: Queue with Deno.kv
+order: 39
+---
+## KV Queue
+
+Queues are a powerful design pattern that help you deal with common application scaling and performance challenges. Some examples of problems that Queues can help you solve are:
+
+Smooth out processing peaks. For example, if users can initiate resource-intensive tasks at arbitrary times, you can add these tasks to a queue instead of performing them synchronously. Then you can have worker processes pull tasks from the queue in a controlled manner. You can easily add new Queue consumers to scale up the back-end task handling as the application scales up.
+Break up monolithic tasks that may otherwise block the Node.js event loop. For example, if a user request requires CPU intensive work like audio transcoding, you can delegate this task to other processes, freeing up user-facing processes to remain responsive.
```
Comment (marco-souza) [inline src/techniques/kvQueue.md]: As we're using Deno, should we change the `Node.js event loop` to something else?
Maybe v8 event loop or JavaScript Event loop 🤔

**Review by marco-souza — APPROVED** [review]
I loved the Dynamic Modules docs! Very well written! 👏👏

File: src/techniques/kvQueue.md (hunk 2 above)
Comment (Sorikairox) [reply src/techniques/kvQueue.md]: Nice catch

**Pull request merged**